README.md

```diff
@@ -1,26 +1,28 @@
 # GitHub OAuth Ruby Quickstart
+
+[DEGW: An intro here motivating the problem you are trying to help solve. Explain (briefly) what an OAuth app is, and why you might want to build one. Why do we need a guide? Is it hard to do? (Narrator: In fact, it is hard.) ALso give a preview of what the reader can expect to have achieved by the end of the article.]
+
+[DEGW: More thoughts. I am certainly guilty of this myself, but I am finding I dislike the word "user" to describe people who will be using your application. We should think about ways to avoid this word in our writing. It will not be easy.]
```

Comment:
Normally I would agree here too. At our last company, we entirely avoided using the word User in our product planning discussions, and it was a good thing. However the issue with that approach here is that we're creating content for developers who are developing any number of different applications. It's impossible for us to give their users a hat to refer to them under. So I agree this will not be easy. I'll think about it, but don't have any ideas at the moment.